modules/deploy/partials/kubernetes/guides/external-access-intro.adoc

```diff
@@ -14,5 +14,5 @@ To access your Redpanda brokers externally, you can map your worker nodes' IP ad
 ====
 IP addresses can change. If the IP addresses of your worker nodes change, you must update your `/etc/hosts` file with the new mappings.
 
-In a production environment, it's best practice to use ExternalDNS to manage DNS records for your brokers. See xref:deploy:deployment-option/self-hosted/kubernetes/kubernetes-best-practices.adoc#use-externaldns-for-external-access[Use ExternalDNS for external access].
+In a production environment, it's best practice to use ExternalDNS to manage DNS records for your brokers. See xref:deploy:deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc#use-externaldns-for-external-access[Use ExternalDNS for external access].
```

Comment:
In a production environment, it's a best practice to use ExternalDNS to manage DNS records for your brokers. See xref:deploy:deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc#use-externaldns-for-external-access[Use ExternalDNS for external access].